pages/broker-assistance/[hackneyId]/assign-package.js

```diff
@@ -27,6 +29,18 @@ const breadcrumbs = [
   { text: 'Assign and attach a care plan' },
 ];
 
+const schema = yup.object().shape({
```

Comment:
Also if you merge latest with your branch you'll see we have a schema file with form schemas in, this should be put there and imported into this page
<img width="1440" alt="Screenshot 2021-11-25 at 11 54 10" src="https://user-images.githubusercontent.com/89010589/143437333-3e1d4062-9695-41a2-a27d-a815946b498f.png">